Sniffs.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Sniffs;
+
+class Sniffs
+{
+    const DEFAULT_POINTS = 30000;
```

Comment:
Since these are a lot of "space after `if`" and etc. I'm not sure most of them would take 5 minutes.

I'll try bumping them up by 40k and seeing how it feels in the app. Thanks.